src/pytest_xray/evidence.py

```diff
@@ -50,3 +53,72 @@ def json(data: AnyStr, filename: str) -> Dict[str, str]:
 
 def zip(data: AnyStr, filename: str) -> Dict[str, str]:
     return evidence(data, filename, APP_ZIP)
+
+
+@pytest.fixture
+def xray_evidence(request) -> Callable:
```

Comment:
Can you add this fixture to `XrayPlugin` class, and remove import `from pytest_xray.evidence import xray_evidence # noqa: F401` from `plugin.py`?
I would like to keep whole business logic in the plugin implementation.